Formula/c/cobo-cli.rb

```diff
@@ -0,0 +1,169 @@
+class CoboCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Cobo Command-line Tool"
+  homepage "https://github.com/CoboGlobal/cobo-cli/blob/master/README.md"
+  url "https://github.com/CoboGlobal/cobo-cli/archive/refs/tags/v0.0.4.tar.gz"
+  sha256 "06903a3c056e38e8898c82601f8abb6bd53633b9de1ee86c8c41cb7e2d03e2ea"
+  license "MIT"
+
+  depends_on "libsodium"
+  depends_on "libyaml"
+  depends_on "python@3.9"
```

Comment:
can it be built with py3.13?

Reply:
yes can . so i need update this to `depends_on "python@3.13"` ?

Reply:
yeah, indeed

Reply:
done